src/libraries/System.Net.Http/tests/StressTests/HttpStress/LogHttpEventListener.cs

```diff
@@ -11,17 +11,20 @@
 using System.Threading.Tasks;
 using System.Runtime.InteropServices;
 using System.Buffers;
+using Microsoft.Extensions.ObjectPool;
 
 namespace HttpStress
 {
     public sealed class LogHttpEventListener : EventListener
     {
+        public const string LogDirectory = "logs";
+
         private int _lastLogNumber = 0;
         private FileStream _log;
         private Channel<string> _messagesChannel = Channel.CreateUnbounded<string>();
         private Task _processMessages;
         private CancellationTokenSource _stopProcessing = new CancellationTokenSource();
-        private const string LogDirectory = "./clientlog";
+        private ObjectPool<StringBuilder> _stringBuilderPool = new DefaultObjectPool<StringBuilder>(new StringBuilderPooledObjectPolicy());
```

Comment:
Not like if this was any major thing, but since we don't really need the abstraction, we can use the specific type & devirtualize the method calls:
```suggestion
        private DefaultObjectPool<StringBuilder> _stringBuilderPool = new DefaultObjectPool<StringBuilder>(new StringBuilderPooledObjectPolicy());
```